dobot_driver/dobot_driver/interface.py

```diff
@@ -97,7 +97,7 @@ def get_alarms_state(self):
 
     def clear_alarms_state(self):
         request = Message([0xAA, 0xAA], 2, 20, True, False, [], direction='out') #BUG
```

Comment:
Ten #BUG nie jest związany ani z metodą `send()`, ani `send_only()`. Chodzi o to, że zgodnie z _Dobot Communication Protocol V1.1.5_ rozkaz  **ClearAllAlarmsState** ma przypisane ID=21. W rzeczywistości ID tego rozkazu to 20. Zidentyfikowałem ten błąd i poinformowałem o nim producenta, lecz od ponad roku nic z tym nie zrobili.